.github/workflows/go.yml

```diff
@@ -29,7 +29,7 @@ jobs:
       - name: check spell
         uses: codespell-project/actions-codespell@master
         with:
-          skip: src,rego-templates,ui
```

Comment:
@elad-da 
What is motivation for the rename? Currently customers are using 'rego-templates' as root folder for their custom templates. Please see for example https://github.com/aquasecurity/postee/blob/main/deploy/kubernetes/postee.yaml#L163
So rename would break existing customer configurations

Reply:
You are right, let me change that back.